server/src/main/java/com/sauron/controller/fake/FakeBankController.java

```diff
@@ -1,29 +1,40 @@
 package com.sauron.controller.fake;
 
-import com.sauron.model.TransactionDto;
+import com.sauron.model.Transaction;
 import com.sauron.model.entities.TransactionDirection;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.math.BigDecimal;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
 import java.util.Collection;
 import java.util.List;
 
 @RestController
 @RequestMapping("/fake/transactions")
 public class FakeBankController {
 
-    @GetMapping("/random-bank")
-    public Collection<TransactionDto> getRandomBankTransactions() {
-        return List.of(new TransactionDto("0700000012348890", TransactionDirection.RECEIVE, new BigDecimal("1000.00")),
-                new TransactionDto("0700000042155212", TransactionDirection.RECEIVE, new BigDecimal("5000.00")),
-                new TransactionDto("0700000099992300", TransactionDirection.PAY, new BigDecimal("400.00")));
+    private Clock clock = Clock.fixed(Instant.parse("2019-06-02T10:15:30.00Z"), ZoneId.of("Europe/Warsaw"));
```

Comment:
You can make it static final and name for example NOW to indicate its role in the test